pkg/npm/npm.go

```diff
@@ -13,14 +17,58 @@ type RegistryOptions struct {
 	SapNpmRegistry     string
 }
 
-type execRunner interface {
+type ExecRunner interface {
 	Stdout(out io.Writer)
 	RunExecutable(executable string, params ...string) error
 }
 
+type NpmUtils interface {
```

Comment:
Should be enough to name this "Utils", as it will be prefixed with "npm." in calling code.